example/lib/ide/ide.dart

```diff
@@ -256,7 +288,7 @@ class BottomBar extends StatelessWidget {
     );
   }
 
-  Widget _buildLspControls() {
+  Widget _buildLspControls(bool isAnalyzing) {
```

Comment:
We don't need this parameter, because we already have a `isAnalyzing` at widget level.